src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java

```diff
@@ -137,33 +136,23 @@
  * 06/12/09 - Alex - Cosmetic changes to GUI
  * -----------------------------------------------------------
  */
-public class AdmittedPatientBrowser extends ModalJFrame implements
-		PatientInsert.PatientListener,
-		PatientInsertExtended.PatientListener, AdmissionBrowser.AdmissionListener,
-		PatientDataBrowser.DeleteAdmissionListener {
+public class AdmittedPatientBrowser extends ModalJFrame implements PatientInsert.PatientListener, PatientInsertExtended.PatientListener,
+				AdmissionBrowser.AdmissionListener, PatientDataBrowser.DeleteAdmissionListener {
 
 	private static final long serialVersionUID = 1L;
 
 	private static final int PANEL_WIDTH = 240;
-	
-	
-	private PatientHistoryManager patientHistoryManager = Context.getApplicationContext().getBean(PatientHistoryManager.class);
 
+	private PatientHistoryManager patientHistoryManager = Context.getApplicationContext().getBean(PatientHistoryManager.class);
 
-	private String[] patientClassItems = {
-			MessageBundle.getMessage("angal.common.all.txt"),
-			MessageBundle.getMessage("angal.admission.admitted.txt"),
-			MessageBundle.getMessage("angal.admission.notadmitted.txt")
-	};
+	private String[] patientClassItems = { MessageBundle.getMessage("angal.common.all.txt"), MessageBundle.getMessage("angal.admission.admitted.txt"),
```

Comment:
I like the old formatting of this array; I find it easier to read and to see what is in the array.

Reply:
I'm using auto-formatting upon save, so the spacing is the one decided by the ide formatter (I don't change anything)

Let me see if it is possible to change the array style...

Reply:
Done (note: the Eclipse profiler version can be update 19 -> 21 in a future change)